src/turner_lab_to_nwb/asap_tdt/asap_tdt_convert_all_sessions.py

```diff
@@ -1,19 +1,26 @@
 import os
+import warnings
 from pathlib import Path
-from typing import Literal
 
-from neuroconv.tools import LocalPathExpander
 from neuroconv.utils import FolderPathType, FilePathType
+from tqdm import tqdm
 
 from turner_lab_to_nwb.asap_tdt.asap_tdt_convert_session import session_to_nwb
 from turner_lab_to_nwb.asap_tdt.utils import load_session_metadata
 
+warnings.filterwarnings("ignore", "Could not identify sev files for channels")
+warnings.filterwarnings(
+    "ignore",
+    r"Complex objects \(like classes\) are not supported\. They are imported on a best effort base but your mileage will vary\.",
+)
+
 
 def convert_sessions(
     folder_path: FolderPathType,
     data_list_file_path: FilePathType,
     output_folder_path: FolderPathType,
-    dataset_mode: Literal["embargo", "public"],
+    gpi_only: bool = False,
```

Comment:
I don't understand why are you passing gpi_only here but then determined it in:
https://github.com/catalystneuro/turner-lab-to-nwb/blob/60053c4c3559cfc1c08fff5795ce860881b902d1/src/turner_lab_to_nwb/asap_tdt/asap_tdt_convert_session.py#L59

Reply:
You're right, it would be more readable to pass it along in `convert_session.py`